zero_bin/rpc/src/main.rs

```diff
@@ -41,6 +47,14 @@ impl Cli {
     /// Execute the cli command.
     pub async fn execute(self) -> anyhow::Result<()> {
         match self {
+            Self::Version {} => {
+                println!(
+                    "Evm Arithmetization package version: {}",
+                    env::var(EVM_ARITH_VER_KEY)?
+                );
+                println!("Build Commit Hash: {}", env::var(VERGEN_RUSTC_COMMIT_HASH)?);
+                println!("Build Timestamp: {}", env::var(VERGEN_BUILD_TIMESTAMP)?);
```

Comment:
You could probably put this code to `print_version` in `zero_bin/common` module, no need to repeat implementation 4 times.